src/fetch.js

```diff
@@ -2,28 +2,28 @@ import fetchSetter from './index.js';
 
 let fetcher = {
  fetchUser(id) {
-    fetch(`https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/travelers/travelers/${id}`)
-        .then(response => response.json())
-        .then(data => fetchSetter.setUserData(data))
-        .then(this.fetchTripsForAUser())
-        .catch(err => console.log('err', err))
+     const fetchedUser = `https://fe-apps.herokuapp.com/api/v1/travel-tracker/data/travelers/travelers/${id}`
+     const promise =  fetch(fetchedUser)
+     .then(response => response.json())
```

Comment:
`.then()` and other dot operators are usually indented when they're on the line below, to more easily show the functionality belongs to the line above.
This is for human eyes. Not computer eyes.